source/apiVolontaria/apiVolontaria/templates/confirm_sign_up.html

```diff
@@ -16,20 +16,20 @@
       </div>
       <div class="email__content">
         <h1 class="email__content__title">
-          {% trans "Bienvenue dans l'équipe NousRire!" %}
+          Bienvenue dans l'équipe NousRire!
         </h1>
         <div class="email_body">
           <p>
-            {% trans "Ton compte bénévole a bien été créé et il ne te reste plus qu'à l'activer:" %}
+            Ton compte bénévole a bien été créé et il ne te reste plus qu'à l'activer:
           </p>
 
-          <p><a href="{{ACTIVATION_URL}}" class="button">{% trans "Activer mon compte" %}</a></p>
+          <p><a href="{{ACTIVATION_URL}}" class="button">Activer mon compte</a></p>
 
-          <p>{% trans "À très bientôt" %}</p>
+          <p>À très bientôt}</p>
```

Comment:
Forgot a `}` at the end of the sentence

Reply:
@RignonNoel Since all the application is in french only for now and all text is written in english, I was seeing no points in having a french text to be translated to english.  If you prefer, I could write a traduction of the textes in english and translate it it french using the translation files.

Reply:
It will be better to write it in english and use the french translation yes. So if one day we want to finally support I18N we don't have to put a big amount of time and risky refactor.

Reply:
@RignonNoel Okay it in place, I have made the best english translation I could ;)